packages/donate-button-next/src/components/widget/components/Footer/index.tsx

```diff
@@ -0,0 +1,25 @@
+import {footerCardCss} from 'src/components/widget/components/Footer/styles';
+import {GridCard} from 'src/components/widget/components/GridCard';
+
+export const Footer = () => {
+	return (
+		<GridCard className={footerCardCss}>
+			<p>
+				100% of your donation is tax-deductible to the extent allowed by US law.
+				Your donation is made to Every.org, a tax-exempt US 501(c)(3) charity
+				that grants unrestricted funds to SDAF on your behalf. As a legal
```

Comment:
To {nonprofit.name} not to SDAF.

Ideally, should replicate the logic we have for `getTaxDeductibleStatement` in the main repo